### PR TITLE
Post release version bump

### PR DESCRIPTION
**Pull request by swapnil-verma-gl**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "alfresco-ng2-components",
   "description": "Alfresco Angular components",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

Comment (MichalKinas) [reply package.json:4]: This should point to next version

File: lib/testing/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-testing",
   "description": "Alfresco ADF testing page and utils",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/testing/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/process-services/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-process-services",
   "description": "Alfresco ADF process services",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/process-services/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/process-services-cloud/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-process-services-cloud",
   "description": "Alfresco ADF process services cloud",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/process-services-cloud/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/js-api/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alfresco/js-api",
-  "version": "8.0.0-alpha.6",
+  "version": "8.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/js-api/package.json:3]:
```suggestion
  "version": "8.0.0-alpha.7",
```

File: lib/insights/package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-insights",
   "description": "Alfresco ADF insights",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/insights/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/extensions/package.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-extensions",
   "description": "Provides extensibility support for ADF applications.",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/extensions/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/eslint-angular/package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alfresco/eslint-plugin-eslint-angular",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/eslint-angular/package.json:3]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/core/package.json [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-core",
   "description": "Alfresco ADF core",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/core/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/content-services/package.json [hunk 10]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-content-services",
   "description": "Alfresco ADF content services",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/content-services/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/cli/package.json [hunk 11]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-cli",
   "description": "Alfresco ADF cli and utils",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
```
Comment (MichalKinas) [inline lib/cli/package.json:4]:
```suggestion
  "version": "7.0.0-alpha.7",
```

File: lib/cli/package.json [hunk 12]
```diff
@@ -1,7 +1,7 @@
...
   "author": "Hyland Software, Inc. and its affiliates",
```
Comment (MichalKinas) [inline lib/cli/package.json:5]: In the same file js-api version should be updated to `"@alfresco/js-api": ">=8.0.0-alpha.7-0",`

File: lib/content-services/package.json (hunk 10 above)
Comment (MichalKinas) [inline lib/content-services/package.json:4]: In the same file update the js-api and adf dependencies version to `8.0.0-alpha.7-0` and `7.0.0-alpha.7-0`

Comment (MichalKinas) [reply lib/content-services/package.json:4]: Same for dependencies in other package.json files

**Review by MichalKinas — CHANGES_REQUESTED** [review]

**Review by MichalKinas — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

File: lib/cli/package.json [hunk 13]
```diff
@@ -20,7 +20,7 @@
     "dist": "rm -rf ../../dist/libs/cli && npm run build && cp -R ./bin ../../dist/libs/cli && cp -R ./resources ../../dist/libs/cli && cp -R ./templates ../../dist/libs/cli && cp ./package.json ../../dist/libs/cli"
   },
   "dependencies": {
-    "@alfresco/js-api": ">=8.0.0-alpha.4",
+    "@alfresco/js-api": ">=8.0.0-alpha.6-0",
```
[the remark this answers is not in the document]
Comment (swapnil-verma-gl) [reply lib/cli/package.json:23]: This is ADF post release bump. The release has already happened

[the remark this answers is not in the document]
Comment (MichalKinas) [reply lib/cli/package.json:23]: Yeah so it should use `>=8.0.0-alpha.7-0` now

[the remark this answers is not in the document]
Comment (swapnil-verma-gl) [reply lib/cli/package.json:23]: I originally used that version only, but was getting failure in GHA. installDeps job on cli project was getting failed. See [here](https://github.com/Alfresco/alfresco-ng2-components/actions/runs/11948718606/job/33312909867#step:4:108)

[the remark this answers is not in the document]
Comment (MichalKinas) [reply lib/cli/package.json:23]: Okay so let's leave it as is then

**Pull request merged**